src/services/doppler-billing-api-client.ts

```diff
@@ -83,6 +96,8 @@ export class HttpDopplerBillingApiClient implements DopplerBillingApiClient {
       dueDate: !!x.dueDate ? new Date(x.dueDate) : null,
       currency: x.currency,
       amount: x.amount,
+      paidToDate: !!x.paidToDate ? x.paidToDate : 0,
+      balance: this.getBalance(x),
```

Comment:
Is it really necessary? I mean, if you do not have the `balance` field you neither have the `paidToDate` field.

So, in my opinion, it should be simply: 

```suggestion
      balance: data.balance == undefined ? x.amount : x.balance
```

or 

```suggestion
      balance: data.balance == undefined ? null : x.balance
```

Reply:
I mean, I think that the backend is responsible for the balance value.

Reply:
Perfect, I like it:

```balance: data.balance == undefined ? x.amount : x.balance```

I will replace for it.

Reply:
ok, please test it, I did not.

Reply:
yes, totally agree

Reply:
@andresmoschini, I will use: ```balance: !!x.balance ? x.balance :  x.amount,``` to be consistent with the other statements. Do you agree?

Reply:
Done